swerve100/src/main/java/org/team100/lib/motor/turning/FalconTurningMotor.java

```diff
@@ -22,23 +23,35 @@ public static class Config {
     public FalconTurningMotor(String name, int canId) {
         m_motor = new WPI_TalonFX(canId);
         m_motor.configFactoryDefault();
+        m_motor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
         m_motor.setNeutralMode(NeutralMode.Brake);
         m_motor.setInverted(InvertType.InvertMotorOutput);
         m_motor.configStatorCurrentLimit(
                 new StatorCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         m_motor.configSupplyCurrentLimit(
                 new SupplyCurrentLimitConfiguration(true, m_config.kCurrentLimit, m_config.kCurrentLimit, 0));
         SmartDashboard.putData(String.format("Falcon Turning Motor %s", name), this);
+        m_motor.configNominalOutputForward(0);
+        m_motor.configNominalOutputReverse(0);
+        m_motor.configPeakOutputForward(1);
+        m_motor.configPeakOutputReverse(-1);
+        m_motor.config_kF(0, 0.05);
```

Comment:
i'm confused by the setting of kF here; kF is the feedforward constant, so it's like you're just adding a small positive constant all the time. why?

Reply:
ah i'm wrong about this, kF is the coefficient.

https://v5.docs.ctr-electronics.com/en/stable/ch16_ClosedLoop.html#how-to-calculate-kf

is that how you came up with 0.05?  that seems to be the number for the zero-load case.  how does it perform with P/I/D zero?